src/Typewriter/Generator.cs

```diff
@@ -96,12 +96,23 @@ static internal string Transform(string csdlContents, Options options)
                 settings.EnableScript = true;
                 transform.Load(options.Transform, settings, null);
 
-                // Execute the transformation, writes the transformed file.
-                transform.Transform(doc, writer);
+                if (options.RemoveAnnotations == false)
```

Comment:
```suggestion
                if (!options.RemoveAnnotations)
```

Reply:
Won't fix. I had it in this form before and I thought the that the terse form was less easy to understand.